packages/components/src/Label/src/Label.module.css

```diff
@@ -0,0 +1,90 @@
+.hop-label {
+    /** TODO: Should all labels be --hop-font-weight-580 instead of the default text */
+
+    /** Component Tokens */
+    --hop-label-component-xs-font-size: var(--hop-body-xs-font-size);
+    --hop-label-component-xs-font-family: var(--hop-body-xs-font-family);
+    --hop-label-component-xs-font-weight: var(--hop-body-xs-font-weight);
+    --hop-label-component-xs-line-height: var(--hop-body-xs-line-height);
+    --hop-label-component-sm-font-size: var(--hop-body-sm-font-size);
+    --hop-label-component-sm-font-family: var(--hop-body-sm-font-family);
+    --hop-label-component-sm-font-weight: var(--hop-body-sm-font-weight);
+    --hop-label-component-sm-line-height: var(--hop-body-sm-line-height);
+    --hop-label-component-md-font-size: var(--hop-body-md-font-size);
+    --hop-label-component-md-font-family: var(--hop-body-md-font-family);
+    --hop-label-component-md-font-weight: var(--hop-body-md-font-weight);
+    --hop-label-component-md-line-height: var(--hop-body-md-line-height);
+    --hop-label-component-lg-font-size: var(--hop-body-lg-font-size);
+    --hop-label-component-lg-font-family: var(--hop-body-lg-font-family);
+    --hop-label-component-lg-font-weight: var(--hop-body-lg-font-weight);
+    --hop-label-component-lg-line-height: var(--hop-body-lg-line-height);
+    --hop-label-component-xl-font-size: var(--hop-body-xl-font-size);
+    --hop-label-component-xl-font-family: var(--hop-body-xl-font-family);
+    --hop-label-component-xl-font-weight: var(--hop-body-xl-font-weight);
+    --hop-label-component-xl-line-height: var(--hop-body-xl-line-height);
+    --hop-label-component-2xl-font-size: var(--hop-body-2xl-font-size);
+    --hop-label-component-2xl-font-family: var(--hop-body-2xl-font-family);
+    --hop-label-component-2xl-font-weight: var(--hop-body-2xl-font-weight);
+    --hop-label-component-2xl-line-height: var(--hop-body-2xl-line-height);
+    --hop-label-component-inherit-size-font-size: inherit;
+    --hop-label-component-inherit-size-font-family: inherit;
+    --hop-label-component-inherit-size-font-weight: inherit;
+    --hop-label-component-inherit-size-line-height: inherit;
+
+}
+
+:where(.hop-label) {
```

Comment:
What is the advantage of using `:where`?

Reply:
it reduces the specificity, so it's easier to overwrite via css.

This is how it was done in Orbiter, but it might not be as necessary in Hopper. 
I'll bring @fraincs in this discussion, as he was the one that introduced the changes in orbiter

Reply:
:where gives a 0 specificity to a declaration

allowing us to easily override "default" styles.

:where(ul[class]) {
  list-style: none;
}

.list {
  list-style: square; /* Now this works like a charm! */
}